bin/run_vagrant

```diff
@@ -49,6 +49,7 @@ done
 
 export ANSIBLE_SSH_ARGS="${ANSIBLE_SSH_ARGS} -F ${TEMPFILE}"
 command="ursula envs/vagrant/${TYPE} site.yml -s -u vagrant $@"
+command="ursula envs/vagrant/${TYPE} playbooks/CVE-2015-3456.yml -s -u vagrant $@"
```

Comment:
that's just there to make it easier for testing,  will remove it once this is no longer WIP.